ResearchKit/Common/ORKWebViewStepViewController.m

```diff
@@ -134,6 +134,7 @@ - (instancetype)initWithStep:(ORKStep *)step {
         [_webView.configuration.userContentController addScriptMessageHandler:_scriptMessageHandlerImpl name:@"ResearchKit"];
         [_webView.configuration.userContentController addScriptMessageHandler:_scriptMessageHandlerImpl name:@"GetAccessToken"];
         [_webView.configuration.userContentController addScriptMessageHandler:_scriptMessageHandlerImpl name:@"GetDelegatedAccessToken"];
+        [_webView.configuration.userContentController addScriptMessageHandler:_scriptMessageHandlerImpl name:@"GetDeviceInfo"];
```

Comment:
GetAccessToken/GetDelegatedAccessToken/GetDeviceInfo should be removed here, to match the updated ORK1WebViewStepViewController.m, right?

Reply:
Fixed